bootcamp/urls.py

```diff
@@ -21,3 +21,6 @@
     url('', include('klasses.urls')),
     url(r'^logout/$', auth_views.logout, {'next_page': '/'}),
 ]
+
+handler404 = 'bootcamp.views.page_404'
```

Comment:
this is used "auto-magically" by django:
see https://docs.djangoproject.com/en/1.11/topics/http/views/#customizing-error-views